ZhukovVA/lab_1/main.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+
+int main() {
+	float num1, num2;
+	char op;
+	int esc;
+
+	do {
+		system("cls");
+		cout << "Type two numbers and an operator in the following format 'Number_1 <Operator> Number_2': ";
+		cin >> num1 >> op >> num2;
+
+		switch (op) {
+		case '+':
+			cout << "Result: " << num1 + num2;;
```

Comment:
Please extract `cout << "Result: " << num1 ? num2;` lines from switch..case in order to reduce code duplication